pkg/reconciliation/reconcile_racks_test.go

```diff
@@ -2952,3 +2952,99 @@ func TestCheckRackLabels(t *testing.T) {
 	require.Equal(result.Continue(), res, "Label updates should not cause errors")
 	require.Subset(desiredStatefulSet.Labels, rc.Datacenter.GetRackLabels("default"))
 }
+
+func TestCheckPodsReadyAllStarted(t *testing.T) {
```

Comment:
Issue: I'm not seeing anything here that actually tests that multiple pods will be restarting at the same time. Is it possible to add this check? I can't manually test for this either (having tried a few times I can't catch it in the act of restarting multiple pods concurrently) so it seems that an envtest will be the only way to confirm that this PR is correct.

Reply:
That is tested in TestStartBootstrappedNodes. This PR isn't about the implementation or functionality of starting bootstrapped nodes, but only about reversing the logic from behind feature flag -> default. I simply added more tests to improve the coverage of scenarios.